client/hack/update-crd.sh

```diff
@@ -39,7 +39,7 @@ then
   exit 1;
 fi
 
-$CONTROLLER_GEN crd:crdVersions=v1,trivialVersions=false paths=${SCRIPT_ROOT}/apis/volumesnapshot/v1
+$CONTROLLER_GEN crd:crdVersions=v1 paths=${SCRIPT_ROOT}/apis/volumesnapshot/v1
```

Comment:
Why is "trivialVersions=false" removed?  Does it cause any problem when running the script?

Reply:
When I run with `v0.8.0`, get the following error:

```
Error: unable to parse option "crd:crdVersions=v1,trivialVersions=false": [unknown argument "trivialVersions" (at <input>:1:31) extra arguments provided: "false" (at <input>:1:32)]
```

Reply:
Sure